igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisService.groovy

```diff
@@ -69,6 +69,17 @@ class TravisService implements BuildService{
         accounts
     }
 
+    void setAccounts() {
+        this.accounts = getAccounts()
+    }
+
+    void syncRepos() {
+        Response response = travisClient.usersSync(getAccessToken())
+        if (response.status != 200) {
```

Comment:
I'm sure retrofit has some method on the `Response` object that will check for success. It might even throw an exception rather than return if there was an error.